source/contributing.md

```diff
@@ -20,6 +20,32 @@ as they help us solve the problem.
 
 You need a [GitHub] account for that.
 
+[github]: https://github.com/
+
+## Write documentation
+
+Documentations on the Nix ecosystem are separated by their purpose.
```

Comment:
Documentation should never be pluralized. In this case, I'd recommend something like: "Documentation on the Nix ecosystem is divided into several domains based on purpose."